.travis.yml

```diff
@@ -11,14 +11,12 @@ matrix:
   # GitHub Pages
   - rvm: 2.4.0
     env:
-    - JEKYLL_VERSION=3.4.3
+    - JEKYLL_VERSION=3.5.2
     - GITHUB_PAGES=1 # Only set on one build in matrix
-  allow_failures:
-  - env: JEKYLL_VERSION=3.5
 env:
   matrix:
-  - JEKYLL_VERSION=3.3.0
-  - JEKYLL_VERSION=3.4.0
+  - JEKYLL_VERSION=3.3
+  - JEKYLL_VERSION=3.4
```

Comment:
I agree that this makes sense, but we will need to update `Gemfile` accordingly, or every build will be using the same version of Jekyll.

Reply:
I'll leave it as it was then, the main goal of this PR was to get rid of the allow failure since https://github.com/jekyll/jekyll/pull/6162 has been fixed.